src/store/rootReducer.ts

```diff
@@ -27,6 +31,8 @@ export enum ReduxActionTypes {
   RESET_GUILD_BOSS_STATS = 'RESET_GUILD_BOSS_STATS',
   LOAD_BATTLE_LOCATIONS = 'LOAD_BATTLE_LOCATIONS',
   REST_BATTLE_LOCATIONS = 'REST_BATTLE_LOCATIONS',
+  LOAD_STAGE_SCORES = 'LOAD_STAGE_SCORES',
+  REST_STAGE_SCORES = 'REST_STAGE_SCORES',
```

Comment:
REST_STAGE_SCORES should be "RESET_STAGE_SCORES" missing an E. I think you copied from above where I made a mistake above with battle locations. Feel free to update that one if you like.

Reply:
Updated RESET_STAGE_SCORES. Will create a separate PR for battle locations.